pkg/commands/kubecfg.go

```diff
@@ -0,0 +1,55 @@
+package commands
+
+import (
+	"fmt"
+	"log"
+	"os/user"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	kubecfg "github.com/ministryofjustice/cloud-platform-cli/pkg/kubecfg"
+)
+
+func addKubecfgCmd(topLevel *cobra.Command) {
+	topLevel.AddCommand(kubecfgCmd)
+	kubecfgCmd.AddCommand(kubecfgShowGithubTeamsCmd)
+
+	kubecfgCommonFlags(kubecfgShowGithubTeamsCmd)
+
+}
+
+var kubecfgCmd = &cobra.Command{
+	Use:   "kubecfg",
+	Short: `Cloud Platform kubeconfig related commands`,
+}
+
+var kubecfgShowGithubTeamsCmd = &cobra.Command{
+	Use:   "show-github-teams",
```

Comment:
I think "id-token" or "id-token-claims" might be a better name for this command.

Reply:
That makes more sense, just did it.